e2e/package.json

```diff
@@ -18,7 +18,17 @@
     "url": "https://github.com/bcgov/alcs/issues"
   },
   "homepage": "https://github.com/bcgov/alcs#readme",
+  "dependencies": {
+    "@playwright/test": "^1.32.0",
+    "@types/node": "^20.11.24",
```

Comment:
nitpick: This doesn't matter since this won't get shipped but types should go in devDependencies

Reply:
Could be. I considered it as a main dependency since running the E2E tests is what this package does. I think you could make the case either way.

Reply:
The transpiled code will be JS not TS, meaning the types won't be used.

Reply:
Good point. Changed.